pkgs/desktops/gnome-3/default.nix

```diff
@@ -156,8 +156,6 @@ lib.makeScope pkgs.newScope (self: with self; {
 
   cheese = callPackage ./apps/cheese { };
 
-  evolution = callPackage ./apps/evolution { };
```

Comment:
You should also delete the old file.

And add alias to the bottom of this one.

Reply:
That file was moved to pkgs/applications/networking/mailreaders/evolution/evolution/default.nix .

It does not show on my system anymore.

Reply:
Weird, GitHub still has that https://github.com/dasj19/nixpkgs/blob/evolution-with-plugins/pkgs/desktops/gnome-3/apps/evolution/default.nix